components/omega/src/infra/logging.h

```diff
@@ -0,0 +1,108 @@
+// Omega logging header file.
+
+#ifndef OMEGA_LOG_H
+#define OMEGA_LOG_H
+
+#pragma once
+
+void omega_log_init();
+
+#include <spdlog/sinks/basic_file_sink.h>
+#include <spdlog/spdlog.h>
+
+#include "YAKL.h"
+
+using yakl::Array;
```

Comment:
All data types should already be defined in DataTypes.h and only Log-specific declarations should be in this header. Use DataTypes.h if these are really neeeded.

Reply:
I will revise Logging.h accordingly.